packages/account/src/Components/self-exclusion/self-exclusion-modal.tsx

```diff
@@ -7,9 +7,9 @@ const SelfExclusionModal = () => {
     const { state, toggleArticle } = React.useContext(SelfExclusionContext);
 
     return (
-        <Modal className='self_exclusion__article-modal' is_open={state.show_article} toggleModal={toggleArticle}>
+        <Modal className='self_exclusion__article-modal' is_open={state?.show_article} toggleModal={toggleArticle}>
             <ThemedScrollbars>
-                <SelfExclusionArticleContent toggleModal={toggleArticle} />
```

Comment:
Don't we need this `toggleModal` function?

Reply:
From what i've checked in the `SelfExclusionArticleContent` there's no props of `toggleModal` inside it and they have their own toggle function calling from `PlatformContext`. I think this toggleModal is not required